docs/source/reference/CLI.md

```
@@ -0,0 +1,329 @@
+
+## zipline init
+
+Initialize a new Zipline project with scaffolding for your cloud provider.
+
+```bash
+zipline init [OPTIONS]
+```
+
+| Option | Description |
+|--------|-------------|
+| `--cloud [gcp\|aws\|azure]` | Cloud provider (required). Also set via `CLOUD_PROVIDER` env var. |
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)